### ISSUE TITLE
ItemJoin server crash during initialization due to clean-up commits.

### ISSUE THREAD

**Issue by mibby** [RockinChaos/ItemJoin#275]
Paper dev 183
Java 14.0.1

ItemJoin dev 403 crashes the server during initialization.

There are no errors, the server just gets stuck initializing at;

```
[01:07:23] [Server thread/INFO]: [ItemJoin] Enabling ItemJoin v5.0.7-SNAPSHOT-b403
```

then detects as crashed and restarts itself automatically.

Downgrading to dev 401, the server starts up fine.

**Comment by RockinChaos** [comment]
Is there any crash log? Using paper build 183 and ItemJoin B403 I am unable to replicate this.

**Comment by mibby** [comment]
There is no crash log unfortunately. The server just restarts immediately when trying to initialize, getting stuck on ItemJoin.

**Comment by RockinChaos** [comment]
Can confirm this issue, looks to have been an issue with the CustomFilter that removes ItemCommand logging (if enabled) but even if it is disabled it still caused immediate crashing.

I have completed my cleanup commits and this issue "seems" to be resolved now. 
Here is the snapshot this was resolved in; http://ci.craftationgaming.com/job/ItemJoin/404/
Be aware this should NOT be used on production servers until it has been further tested.

**Comment by mibby** [comment]
Seems to be confirmed fixed with dev 411. Server doesn't freeze or crash at start-up initialization. Also no issues during shutdown.

**Comment by RockinChaos** [comment]
This bug has been officially fixed in the release of ItemJoin v5.0.7.
You can grab it here; Download: https://www.spigotmc.org/resources/itemjoin.12661/download?version=354306

Please read the changelog as the documentation has not been updated yet;
https://github.com/RockinChaos/ItemJoin/wiki/Recent-Changes
